pkgs/games/prismlauncher/default.nix

```diff
@@ -41,8 +43,15 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-sKAhcbDoRbWf/DuwcBmDjb+VSMM0K2C33gu1K9AlPoQ=";
   };
 
-  nativeBuildInputs = [ extra-cmake-modules ghc_filesystem cmake file jdk wrapQtAppsHook ];
-  buildInputs = [ qtbase qtsvg zlib quazip tomlplusplus ];
+  nativeBuildInputs = [ extra-cmake-modules cmake file jdk wrapQtAppsHook ];
+  buildInputs = [
+    qtbase
+    qtsvg
+    zlib
+    quazip
+    ghc_filesystem
```

Comment:
just a small nit: this is a build-time only dependency (it's a header only library), so it actually should be in `nativeBuildInputs`

Reply:
As I know header-only libraries should be in `buildInputs`.

Reply:
This was discussed a in another PR as well. Not sure how to handle this exactly https://github.com/NixOS/nixpkgs/pull/196624#discussion_r998594889

Reply:
Actually it doesn't matter since it's just a header-only library. I looked at a few packages and all of them handled header-only libraries in `buildInputs`. Let's just follow rest of nixpkgs.

Reply:
Hmm, yeah I'd expect anything that doesn't end up as a runtime input to be in `nativeBuildInputs`, but it seems like the rest of nixpkgs is pretty inconsistent about this, so I think this is ok!

I suppose it's possible to have a package that generates different headers based on platform. I doubt that's common, but I could see why you'd prefer `buildInputs` in those cases, and then prefer it in general when you're not sure.

Reply:
lol I think I convinced myself that because of this, only compilers should ever go in `nativeBuildInputs`. I wish there was more information on this. I could only find people saying they think it should go in `buildInputs` I couldn't actually find any info on why.